README.md

```diff
@@ -27,19 +27,19 @@ var sodium = require('sodium-native')
 var nonce = new Buffer(sodium.crypto_secretbox_NONCEBYTES)
 var key = sodium.sodium_malloc(sodium.crypto_secretbox_KEYBYTES) // secure buffer
 var message = new Buffer('Hello, World!')
-var cipher = new Buffer(message.length + sodium.crypto_secretbox_MACBYTES)
+var ciphertext = new Buffer(message.length + sodium.crypto_secretbox_MACBYTES)
```

Comment:
we should change these to Buffer.alloc/from now (diff pr)

Reply:
Agree

Reply:
Fixed in https://github.com/sodium-friends/sodium-native/commit/678fee7439ba18aad2d96a31cb1b2bb782aa4458